scripts/create_perf_json.py

```diff
@@ -1,3 +1,7 @@
+# Copyright (C) 2022 Intel Corporation
+# Copyright (C) 2022 Google
```

Comment:
For Google authored code, the required Google style for copyright notices is:
Copyright 2022 Google LLC
This work has been joint work started by Intel, so using Intel's style I think is okay but maybe add the LLC.

Reply:
Thanks Ian. I added "LLC" in https://github.com/intel/perfmon/pull/152/commits/ef448c779b264fa0418211707b571bce510776a3 . The stock Intel template included `(C)`. I'm not we need to make the two templates match exactly. Would it look strange to have one line with (C) and another without?

Reply:
Given this isn't Google authored code, not strictly adhering to Google's style I think is okay.  So I'd keep the "(C)" for consistency. Thanks!